roles/varnish/tasks/main.yml

```diff
@@ -48,3 +48,6 @@
     group: root
     mode: 0644
   notify: Reload varnish
+
+- include_role:
```

Comment:
If we're going to include the `varnish_exporter` role here why is it a separate role to begin with? It could be a separate task that gets included here. If there is a good reason to make it a separate role it should be imported in the playbook.

Reply:
good call, will change :+1: